elf2uf2/elf2uf2.cpp

```diff
@@ -204,10 +304,12 @@ int bin2uf2(std::shared_ptr<std::iostream> in, std::shared_ptr<std::iostream> ou
     if (size <= 0) {
         fail_read_error();
     }
+    in->seekg(0, in->beg); // Reset to beginning
```

Comment:
This isn't actually required, as the function that reads the file (`realize_page`) seeks to the appropriate place in the file before reading